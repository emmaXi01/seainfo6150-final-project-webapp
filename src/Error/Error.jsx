import React from "react";
import styles from "./Error.module.css";

const Error = () => {
  return (
    <div className={styles.error}>
        404 Not Found! <br/> <br/>The error page!
    </div>
  );
};

export default Error;
