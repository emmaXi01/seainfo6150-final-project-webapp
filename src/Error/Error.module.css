.error {
  display: block;
  text-align: center;
  font-weight: 700;
  font-size: 25px;
  padding: 50px;
  margin: 50px; 
}