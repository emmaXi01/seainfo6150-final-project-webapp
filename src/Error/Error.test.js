import React from "react";
import { render } from "@testing-library/react";
import Error from "./Error.jsx";

describe("Error test", () => {
  it("renders the error component correctly", () => {
    const { container } = render(<Error />);
    expect(container).toMatchSnapshot();
  });
});