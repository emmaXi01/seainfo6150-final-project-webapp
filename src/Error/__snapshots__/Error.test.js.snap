// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error test renders the error component correctly 1`] = `
<div>
  <div
    class="error"
  >
    404 Not Found! 
    <br />
     
    <br />
    The error page!
  </div>
</div>
`;
